src/components/Footer/MailchimpForm.tsx

```diff
@@ -36,48 +30,20 @@ const CustomForm: FC<FormHooks<EmailFormFields>> = ({
           Receive the latest news about Threshold
         </BodyLg>
       </SectionTextContainer>
-      <Box
-        as="form"
-        // @ts-ignore
-        onSubmit={handleSubmit}
-        w="full"
-        maxW={{ base: "100%", md: "320px", lg: "512px" }}
-      >
-        <FormControl>
-          <ResponsiveStack columnReverse>
-            <Input
-              size="lg"
-              onChange={(e) => setEmail(e.target.value)}
-              type="email"
-              value={email}
-              bg="white"
-              placeholder="Enter an email"
-            />
-            <Button
-              size="lg"
-              px={6}
-              type="submit"
-              isLoading={status === "sending"}
-            >
-              Submit
-            </Button>
-          </ResponsiveStack>
-
-          {status === "error" && (
-            <FormHelperText
-              color="red.500"
-              // slice first 4 chars to scrub the mailchimp error
-              // example mailchimp error: "0 - Invalid Email"
-              dangerouslySetInnerHTML={{ __html: (message as string).slice(4) }}
-            />
-          )}
-
-          {status === "success" && (
-            <FormHelperText color="green.500">
-              Please check your email to verify and join the newsletter.
-            </FormHelperText>
-          )}
-        </FormControl>
+      <Box w="full" maxW={{ base: "100%", md: "320px", lg: "512px" }}>
+        <iframe
+          src="https://embeds.beehiiv.com/38f40972-ced2-4f8e-b92b-2030750a59d0?slim=true"
+          data-test-id="beehiiv-embed"
+          height="52"
+          width="100%"
+          frameBorder="0"
+          scrolling="no"
+          style={{
+            margin: 0,
+            borderRadius: "0",
+            backgroundColor: "transparent",
+          }}
+        ></iframe>
```

Comment:
Let's use `<iframe />` instead `<iframe></iframe>`- there are no children.

Reply:
Fixed in 728c282e33e7d69c0017dac9e73efee6c7486701